tests/io/test_read_eea_aqerep_v2.py

```diff
@@ -1,26 +1,23 @@
 import pytest
 
-from pyaerocom.io import ReadUngridded
 from pyaerocom.stationdata import StationData
 from pyaerocom.ungriddeddata import UngriddedData
+from tests.conftest import TEST_RTOL
 
-# although the following is not explicitly referenced, it registers the
-# Subset data ids used for testing.
-
+# Subset data id used for testing.
 DATA_ID = "EEA_AQeRep.v2.Subset"
 
 
 @pytest.fixture(scope="module")
-def reader():
-    return ReadUngridded(DATA_ID)
+def reader(eea_v2_subset_reader):
+    return eea_v2_subset_reader
```

Comment:
Why is this fixture needed?
If all you want is to rename the rename the fixture you can import it with the new name:

``` python
from tests.fixtures.eea_v2 import eea_v2_subset_reader as reader
```